Automattic/jetpack
# Fix Verbum Block Editor translations

## Proposed changes:
This will enqueue the translations available in `widgets.wp.com` for Verbum Block Editor.


### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?
No

## Testing instructions:
1. Sync jetpack-mu-plugin to your sandbox using `rsync`.
2. You'll also need [this Calypso PR](https://github.com/Automattic/wp-calypso/pull/90686). Please check it out locally, in package/verbum-block-editor, run yarn dev --sync.
3. Sandbox punsintended418746564.wordpress.com.
4. Visit https://punsintended418746564.wordpress.com/2024/02/06/blogpost-broken-bravtar/.
5. Editor should be translated.
